docs/release-train-issue-template.md

```diff
@@ -0,0 +1,73 @@
+Release Akka HTTP $VERSION$
+
+<!--
+Release Train Issue Template for Akka HTTP
+
+(Liberally copied and adopted from Scala itself https://github.com/scala/scala-dev/blob/b11cd2e4a4431de7867db6b39362bea8fa6650e7/notes/releases/template.md)
+
+For every release, use the `scripts/create-release-issue.sh` to make a copy of this file named after the release, and expand the variables.
+
+Variables to be expanded in this template:
+- VERSION=???
+
+Key links:
+  - akka/akka-http milestone: https://github.com/akka/akka-http/milestone/?
+-->
+
+### Cutting the release
+
+- [ ] Check that open PRs and issues assigned to the milestone are reasonable
+- [ ] For minor or major versions, update the Change date in the LICENSE file and update the `licenses` url in the build.
+- [ ] Create a PR to add a release notes entry in docs/src/main/paradox/release-notes/. As a helper run `scripts/commits-for-release-notes.sh <last-version-tag>` which will output a list of commits grouped by submodule, and the closed issues for this milestone
```

Comment:
So we keep this special case for Akka HTTP for now rather than completely align cross projects?

Reply:
I agree, it feels like a cumbersome step to me.